spec/models/box_request_spec.rb

```diff
@@ -122,9 +122,8 @@
       box_request.reviewed_by_id = user.id
       box_request.claim_review
       box_request.reviewed_at = Time.now
-      @box = Box.create(box_request_id: box_request.id)
       box_request.complete_review
-      expect(box_request.box).to have_state(:reviewed)
+      expect(box_request.box).to have_state(:reviewed);
```

Comment:
we don't do `;` :)

Reply:
That will need to be done in a new PR. 
Will accept this one as is. 
Thanks so much for your help!